### DISCUSSION TITLE
Twitter crawler might stop working soon!

### DISCUSSION THREAD

**Discussion by thomas694** [TumblThreeApp/TumblThree#430] [General]
"Starting February 9, we will no longer support free access to the Twitter API, both v2 and v1.1." [https://twitter.com/TwitterDev/status/1621026986784337922]

Well then? ...

**Comment by thomas694** [comment]
For now they have blocked public access to tweets. As a result the blogs are shown offline.

**Reply by Linktydraa** [reply]
I think the public access is back. I've rechecked some handles that were previously marked as offline, and it went back to being online.
